elements/glossy-portfolio-theme/lib/glossy-portfolio-grid.js

```diff
@@ -128,52 +177,61 @@ export class GlossyPortfolioGrid extends DDDSuper(I18NMixin(LitElement)) {
          gap: 25px;
 
         }
-        .container-background{
-          padding: var(--mobile-page-padding);
 
-        }
       }
 
     `];
   }
 
   // Lit render the HTML
   render() {
+return html`
+${this.data.length > 0 ?   html`        
+  <div class = "container-background">
+    <div class="projects-header">
 
-    return html`
-          
-<div class = "container-background">
-  <div class="projects-header">
+      <div class="grid-title">${this.title.toUpperCase()}</div>
+      <div class="filters">
+        ${this.displayFilters()}
+      </div>
+
+    </div>
+    <div class="card-container">
+      ${this.filteredData.map((item)=>{ return html`
+          <glossy-portfolio-card class="card" 
+          title="${item.title}" 
+          thumbnail=${item.metadata.images[0]?
+          item.metadata.images[0]
+          :store.manifest.metadata.site.logo}
+          slug="${item.slug}"
+          >
+        </glossy-portfolio-card>
+        `})}
+      </div> 
+  </div> `
+  :  html``
+  }
+`;}
 
-    <div class="grid-title">${this.title.toUpperCase()}</div>
-    <div class="filters">
+  displayFilters() {
+    //display/print filters (top left of the grid)
+
+    if(this.filtersList.length === 0){
+      // hide "All" button if there is no filter
+      return html`   
+       <button style = "display: none" class="filter active" name="all" @click="${this.updateFilter}">All</button>
```

Comment:
just don't render anything if there's no button and your just display none'ing it anyway